active_admin-sortable_tree.gemspec

```
@@ -5,25 +5,33 @@ require "active_admin/sortable_tree/version"
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "active_admin-sortable_tree"
-  s.version     = ActiveAdmin::SortableTree::VERSION
-  s.authors     = ["Francesco Disperati", 'Charles Maresh']
-  s.email       = ["me@nebirhos.com", 'zorab47@gmail.com']
-  s.homepage    = "https://github.com/zorab47/active_admin-sortable_tree"
-  s.summary     = "Add drag and drop sorting to ActiveAdmin resources"
+  s.name = "active_admin-sortable_tree"
+  s.version = ActiveAdmin::SortableTree::VERSION
+  s.authors = ["Effy Tech"]
+  s.email = ["technique@effy.fr"]
+
+  s.summary = "Add drag and drop sorting to ActiveAdmin resources"
   s.description = "SortableTree provides sorting of lists and hierarchies from ActiveAdmin index views."
-  s.license     = "MIT"
+  s.license = "MIT"
+
+  s.homepage = 'https://github.com/effy-tech/active_admin-sortable_tree'
+  s.metadata['allowed_push_host'] = 'https://rubygems.pkg.github.com/effy-tech'
+  s.metadata['homepage_uri'] = s.homepage
+  s.metadata['source_code_uri'] = 'https://github.com/effy-tech/active_admin-sortable_tree'
+  s.metadata['changelog_uri'] = 'https://github.com/effy-tech/active_admin-sortable_tree'
+
+  s.files = `git ls-files app lib vendor`.split($\) + ["Changelog.md", "README.md", "MIT-LICENSE"]
 
-  s.files       = `git ls-files app lib vendor`.split($\) + ["Changelog.md", "README.md", "MIT-LICENSE"]
+  s.required_ruby_version = Gem::Requirement.new
 
-  s.add_dependency 'activeadmin', '>= 1.1'
-  s.add_dependency 'coffee-rails'
-  s.add_dependency 'jquery-rails'
-  s.add_dependency 'sass',        '~> 3.1'
+  s.add_dependency('activeadmin', '~> 1.1')
```

Sinon ça casse suite à la mise à jour par Alexandre d'ActiveAdmin à 2.7 ...
```suggestion
  s.add_dependency('activeadmin', '>= 1.1')
```

---

heu... ok sur le principe, pas trop fan du '>=', mais je veux pas n'importe quel version sup a 1.1... c'est pas safe. 😉

---

J'aurais bien fait une autre PR du coup car celle ci cherche a mep la v 0.0.6. si on update les requirements, il faudrais bump la version de la gem et du package... a mon avis.

---

Ben dans ce cas on fait 
```suggestion
  s.add_dependency('activeadmin', '~> 2.7')
```

---

> J'aurais bien fait une autre PR du coup car celle ci cherche a mep la v 2.0.0. si on update les requirements, il faudrais bump la version de la gem et du package... a mon avis.

OK.